hedging/src/job/adjust_funding.rs

```diff
@@ -0,0 +1,222 @@
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
+use tracing::instrument;
+
+use galoy_client::*;
+use okex_client::*;
+use shared::pubsub::CorrelationId;
+
+use crate::{error::*, okex_transfers::*, rebalance_action::*, synth_usd_liability::*};
+
+use super::HedgingFundingConfig;
+
+const SATS_PER_BTC: Decimal = dec!(100_000_000);
+
+#[instrument(name = "adjust_funding", skip_all, fields(correlation_id = %correlation_id,
+        target_liability, current_position, last_price_in_usd_cents, funding_available_balance,
+        trading_available_balance, onchain_fees, action, external_client_transfer_id, internal_client_transfer_id,
+        transferred_funding) err)]
+pub(super) async fn execute(
+    correlation_id: CorrelationId,
+    synth_usd_liability: SynthUsdLiability,
+    okex: OkexClient,
+    okex_transfers: OkexTransfers,
+    galoy: GaloyClient,
+    hedging_funding_config: HedgingFundingConfig,
```

Comment:
I think instead of passing config in here we should pass `AdjustFunding` or `AdjustHedging` structs around. I will explain more below.